content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -0,0 +1,116 @@
+
+![Generate XRP Ledger Devnet credentials](img/evm-sidechain-xrpl-devnet-faucet.png "Generate XRP Ledger Devnet credentials")
+
+Note the address and secret associated with your Devnet address. You will need this information to set up your preferred XRP Ledger wallet. 
+
+## 3. Transfer Funds Using the EVM Sidechain Bridge
+
+Once you have your accounts setup and test fund allocated, you can use the EVM Sidechain bridge to move the test XRP tokens to the EVM Sidechain. 
+
+The EVM Sidechain bridge is a tool that allows the user to transfer funds between chains in a fast and secure way.
+
+To start using the bridge, go to [https://bridge.devnet.xrpl.org](https://bridge.devnet.xrpl.org/)
+
+### 1. Connect Both Wallets
+
+**Connect Xumm Wallet**
+
+The Xumm wallet is used to interact with the XRP Ledger Devnet chain. 
+Ensure that you have created an account on the public XRP Ledger Devnet as described in Step 1. 
+
+To connect a Xumm wallet to the bridge, go to the [EVM Sidechain bridge](https://bridge.devnet.xrpl.org) and click “Connect with Xumm Wallet”. 
+
+![Connect XUMM Wallet](img/evm-sidechain-connect-xumm-wallet.png "Connect XUMM wallet")
+
+
+**Note:** Ensure that you are connected to XRP Ledger Devnet and that the application that you are connecting with is the correct one.
+
+Follow the instructions on screen to scan the QR code using the Xumm app. The Xumm wallet app will display a confirmation page similar to the following image.
+
+![Connect to XRP Ledger Devnet](img/evm-sidechain-bridge-sign-in.jpg "Connect to XRP Ledger Devnet")
+
+**Connect MetaMask Wallet**
+
+The MetaMask wallet is used to interact with the XRP Ledger EVM Sidechain. 
+Ensure that you have created a MetaMask account and connected to the public XRP Ledger Devnet as described in [Connect MetaMask to XRP Ledger EVM Sidechain](connect-metamask-to-xrpl-evm-sidechain.html).
+
+To connect a MetaMask wallet to the bridge, go to the [EVM Sidechain bridge](https://bridge.devnet.xrpl.org) and click “Connect with Metamask Wallet”.
+
+![Connect MetaMask Wallet](img/evm-sidechain-connect-metamask.png "Connect MetaMask wallet")
+
+### 2. Initiate the Transfer of Funds 
+
+Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount to send, and the destination address.
+
+- **Direction of the bridge**: This is the direction of transfer; it can be either EVM sidechain → XRP Ledger Devnet or XRP Ledger Devnet → EVM sidechain. Use the “Switch Network” button to switch the direction of transfer.
+- **Amount to send**: This is the amount that you want to transfer to the other side. Note that there is a fee to use the bridge.
+    - Network fees: The fees that will go to the network by using the transactions needed for the operation.
+    - Commission: The bridge applies a commission for every transaction completed and is to prevent spam and DDOS.
```

Comment:
...completed. This is to prevent spam and distributed denial of service attacks.